lib/diagnostic.js

```diff
@@ -38,19 +49,41 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+      name: 'Dewey',
+      species: 'Dog',
+      breed: 'Lakeland Terrier',
+      vocalization: 'bark',
+      vocalize: function(){
+        return 'bark';
+      }
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.vocalization = vocalization;
 };
 
+
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+const Pet = function (name, species, breed, vocalization) {
```

Comment:
```Pet``` was declared earlier with ```const```.  This will throw an error since ```Pet``` is supposed to be a constant and it's being redeclared/redefined here.